src/Combinatorics/Graphs/functions.jl

```diff
@@ -254,6 +454,10 @@ function add_vertices!(g::Graph{T}, n::Int64) where {T <: Union{Directed, Undire
   return count(_->add_vertex!(g), 1:n)
 end
 
+function add_vertices!(mg::MixedGraph, n::Int64)
+  return (add_vertices!(directed_component(mg), n) +
+    add_vertices!(undirected_component(mg), n)) // 2
```

Comment:
Could you please explain where the `// 2` comes from? If this was about edges, I would totally understand it, but this function is about vertices.

Reply:
Ah, I missed the parentheses. But shouldn't the function check that both are the same somehow?

Reply:
I think this is related to https://github.com/oscar-system/Oscar.jl/pull/5179#discussion_r2272898392.

My only real comment here is that I think we should return an Int here and currently return a Rational

Reply:
yes i agree with your comment, I am about to push some changes where i create copies, and I'll fix return type here